### ISSUE TITLE
php artisan heimdall:add --url http://nextclout.cgt.fr  --title nextcloud --type nexcloud  

### ISSUE THREAD

**Issue by Benouare** [linuxserver/Heimdall#920]
Hi, 

I would like to be able to "add" items with something like this 
php artisan heimdall:add --url http://nextclout.cgt.fr  --title nextcloud --type nexcloud  
is it currently possible? 
I dont want to use an api or to do it manually. 

Thx & cheers

**Comment by KodeStar** [comment]
No, not currently, I'll move this to ideas though and if it gets enough upvotes I'll have a look
